### PR TITLE
DL-7585 Modified AddDetailsView

### PR DESCRIPTION
**Pull request by rajanimohan22**
# JIRA Story DL-7585 -Upgrade views to Govuk frontend

New feature

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist

 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/views/AddDetailsView.scala.html [hunk 1]
```diff
@@ -19,57 +19,64 @@
...
-        @components.back_link()()
+    @if(form.errors.nonEmpty) {
+        @errorSummary(ErrorSummary().withFormErrorsAsText(form))
+    }
 
-        @components.error_summary(form.errors)
+    @formWithCsrf(action = PDFController.onSubmit(mode), 'autoComplete -> "off") {
 
-        @components.heading("addDetails.heading")
+        @heading("addDetails.heading")
```
Comment (benryan1790) [inline app/views/AddDetailsView.scala.html:47]: Can we use the headingNew component here as it contains the correct classes while the heading component uses the old play-ui classes

File: app/views/components/input_text_new.scala.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+@import config.FrontendAppConfig
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.label.Label
+
+@this(inputText: GovukInput)
+@(
+        id: String = "",
+        name: String,
+        field: Field,
+        hintKey: Option[String] = None,
+        inline: Boolean = false,
```
Comment (benryan1790) [inline app/views/components/input_text_new.scala.html:29]: the inline is not needed as the `govuk-radios--inline` class relates specifically to radio button inputs

File: app/views/components/input_textarea_new.scala.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@*
+* Copyright 2023 HM Revenue & Customs
```
Comment (benryan1790) [inline app/views/components/input_textarea_new.scala.html:18]: Please delete the duplicated copyright notice. If you delete both and then recompile it should generate the notice for you.

**Review by benryan1790 — CHANGES_REQUESTED** [review]

File: app/views/AddDetailsView.scala.html (hunk 1 above)
Comment (rajanimohan22) [reply app/views/AddDetailsView.scala.html:48]: Done

**Review by benryan1790 — APPROVED** [review]

**Pull request merged**